tests/phpunit/tests/theme/themeDir.php

```diff
@@ -185,6 +185,7 @@ public function test_theme_list() {
 			'Block Theme [0.4.0]',
 			'Block Theme [1.0.0] in subdirectory',
 			'Block Theme Deprecated Path',
+			'Block Theme Hooked Blocks',
```

Comment:
Nitpick, the name is a bit clunky (especially when read as a path name, `block-theme-hooked-blocks`). How about something shorter, e.g. `Hooked Blocks Theme`? 🤔 (Although I realize we might want to have it next to the other `Block Theme ...` test themes, so maybe not a good idea 😅 )

Reply:
I mostly wanted to have the same prefix:

<img width="429" alt="Screenshot 2023-10-02 at 18 02 04" src="https://github.com/WordPress/wordpress-develop/assets/699132/0b4ff2b1-7a66-40e9-851d-72b09e412854">

It could also become `Block theme with hooked blocks`. I'm open to changing it.

Reply:
Trying `block-theme-with-hooked-blocks` and `Block Theme with Hooked Blocks` in https://github.com/WordPress/wordpress-develop/pull/5340/commits/2caf1146e8e7706b166e9325aa86399b1182bd9b.